src/lib.rs

```diff
@@ -423,6 +426,43 @@ impl CommonMarkViewer {
         response
     }
 
+    /// Shows rendered markdown, and allows the rendered ui to mutate the source text.
+    ///
+    /// The only currently implemented mutation is allowing checkboxes to be toggled through the ui.
+    pub fn show_mut(
```

Comment:
You suggested `show_mutable` as the name, but I think `show_mut` is more idiomatic for a Rust API name.